### DISCUSSION TITLE
start with activated vinyl control

### DISCUSSION THREAD

**Discussion by molli1312** [fayaaz/mixxx-pi-gen#91] [Q&A]
Hello, I'm looking for a way to start Mixxx with activated vinyl control.

I changed vinylcontrol 0 to 1 in /.mixxx/mixxx.cfg. But to no avail.

mixxx& sleep 10&& xdotool key Ctrl+t unfortunately doesn't produce the desired result on the PI.

Does anyone have an idea.
Feedback geben

**Comment by fayaaz** [comment]
If you activate it and then quit and reopen it does it not reopen activated? You could write a custom controller mapping I guess to set this when it loads up.

xdotool doesn't work under wayland. Have a look at [ydotool](https://gabrielstaples.com/ydotool-tutorial/#gsc.tab=0)
